dotnet/src/SemanticKernel.UnitTests/CoreSkills/MathSkillTests.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Are these skills required for the conditional feature?

Reply:
Unless they are required or we have commitment to adding these to the kernel, I'd suggest keeping them as either test/example skills.

Reply:
They aren't

Reply:
cc @shawncal for advice on where these skills belong.

Reply:
Sure, I have some good Unit Tests on those and would be good to have one place (if not in CoreSkills) that they could be tested and used for Integration Tests and Samples.